File: Easy_problems/FindClosestNumberToZero_2239/Sol.java
public class Sol {
    public static void main(String[] args) {
        System.out.println(findClosestNumber(new int[]{-1, 2, 1}));
    }

    public static int findClosestNumber(int[] nums) {
        int curr = Integer.MAX_VALUE;

        for(int num : nums)
            if (Math.abs(num) < Math.abs(curr) || (Math.abs(num) == Math.abs(curr) && num > curr))
                curr = num;

        return curr;
    }
}

File: Easy_problems/FindClosestNumberToZero_2239/Sol.py
class Solution():
    def findClosestNumber(self, nums):
        current = 99999999
        
        for num in nums:
            if abs(num) < abs(current):
                current = num
            elif abs(num) == abs(current) and num > current:
                current = num 
            
            
        return current
        